src/tools/gravity-parseList.c

```diff
@@ -95,9 +95,9 @@ inline bool __attribute__((pure)) valid_domain(const char *domain, const size_t
 		return false;
 
 	// TLD length check
-	// The last label must be at least 2 characters long
+	// The last label must be at least one character long
```

Comment:
Maybe we could add the issue number here, to help us understand in the future why this is different then the expected.

```suggestion
	// The last label must be at least one character long (see FTL #1920)
```

Reply:
We have never done this before and I don't think it's good practice. You can always use `git blame` to get context. Otherwise, it's difficult to maintain such links when stuff gets rewritten/extended in the future and references may become outdated

Reply:
I agree the issue number is not the best idea.

I just think we should have a better comment explaining the text _"at least one character long"_, since this is not what we expect for TLD and we might forget the reason why this was set.

Anyway, it's just a suggestion.

Reply:
The check and comments were removed.

Explanation (in your words):
"We already check for empty labels and every label that is non-empty has at least one character in it".